src/client/containers/BoardPageComponents/BoardAddCandidate/BoardHeader.style.css

```diff
@@ -0,0 +1,32 @@
+* {
+  box-sizing: border-box;
+}
+
+.board-header-container {
+  position: relative;
+  height: 739px;
+  max-width: 414px;
+}
+
+.header-component-image {
```

Comment:
board- prefexi here also

Reply:
this class comes directly from the Header component, I couldn't change it. The same with dropdown button

Reply:
If the class is already used in the header component then it shouldn't be set here also. You will have to make a change to the header component then. Maybe make it more flexible

Reply:
Niels but the HeaderComponent is already merged, how can I make changes there? Should I open a new PR for it? Basically the background image came from there non cropped and the only way I could adjust it to fit to the screen and be harmonised with the other components of the BoardPage was through this class.

Reply:
You can make changes to the HeaderComponent from here if you want. You should rely on the css in the header compoent only. Otherwise you can make it more flexible by inserting props of by making it more responsive. Overwriting css classes like you are doing now is considered a hacky solution